hexrd/fitting/calibration/calibrator.py

```diff
@@ -38,7 +36,6 @@ def update_from_lmfit_params(self, params_dict):
         For example, the powder calibrator will update the lattice parameters
         on the material. The Laue calibrator will update crystal parameters.
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.